ale/drivers/dawn_drivers.py

```diff
@@ -152,3 +123,27 @@ def starting_ephemeris_time(self):
             self._starting_ephemeris_time = spice.scs2e(self.spacecraft_id, sclock)
             self._starting_ephemeris_time += 193.0 / 1000.0
         return self._starting_ephemeris_time
+
+    @property
+    def _odtk(self):
```

Comment:
@jessemapel I imagine this needs to be public as well?

Reply:
This is not part of the base Driver interface so it can be private. If you look at #144 it accesses this via `_odtk`